README.md

```diff
@@ -1 +1,19 @@
 <p align="center"><img src="https://user-images.githubusercontent.com/8326331/133301616-504b30a2-a015-4de8-b611-60539f133cfc.png" height="200px" /></p>
+
+## Development
+
+### Setup
+
+Install [Node](https://nodejs.org/) and [Yarn](https://yarnpkg.com/) if you haven't already.
```

Comment:
Do we want to add anything about NVM and node versions?

Reply:
It looks like GH actions is using node 14, which is the latest stable release, so we might put something in about using at least v14.x to match our build process. NVM seems a bit outside the scope of this README to me, but if you both think it's helpful I don't think it'd hurt.

Reply:
I normally wouldn't include this, but I'll add a note.

Reply:
I'm out of the loop on node and versioning, so if you think it's not needed I trust your judgement